### PR TITLE
Simplify Azure blobStore method signatures

### PR DESCRIPTION
**Pull request by dadoonet**
While working on #26751, I found that we are passing the container name on every single method although we don't need it as it is stored within the blobstore object already.

This PR simplifies a bit that part of the code.

### CODE REVIEW

File: plugins/repository-azure/src/main/java/org/elasticsearch/repositories/azure/AzureBlobStore.java [hunk 1]
```diff
@@ -99,42 +95,42 @@ public void delete(BlobPath path) {
...
     }
 
-    public boolean doesContainerExist(String container)
+    public boolean doesContainerExist()
     {
         return this.client.doesContainerExist(this.clientName, this.locMode, container);
     }
 
-    public void deleteFiles(String container, String path) throws URISyntaxException, StorageException
+    public void deleteFiles(String path) throws URISyntaxException, StorageException
```
Comment (imotov) [inline plugins/repository-azure/src/main/java/org/elasticsearch/repositories/azure/AzureBlobStore.java]: I don't think this method is used anymore. So, maybe you should clean it up as well while you are at it.

**Review by imotov — APPROVED** [review]
LGTM.

**Review by imotov — APPROVED** [review]
Still LGTM

**Comment by dadoonet** [comment]
@elasticmachine test it please

**Comment by dadoonet** [comment]
Thanks @imotov.

I pushed it in master and in 6.x with 6cb6035.

**Pull request merged**